### PR TITLE
Bug 1782053 - [PerfCompare] Calculate is_confident, needs_more_runs, is_noise_metric

### PR DESCRIPTION
**Pull request by beatrice-acasandrei**

### CODE REVIEW

File: treeherder/webapp/api/performance_serializers.py [hunk 1]
```diff
@@ -458,6 +458,9 @@ class PerfCompareResultsSerializer(serializers.ModelSerializer):
     delta_percentage = PerfCompareDecimalField()
     magnitude = PerfCompareDecimalField()
     new_is_better = OptionalBooleanField()
+    is_confident = OptionalBooleanField()
+    more_runs_are_needed = OptionalBooleanField()
+    noise_metric = OptionalBooleanField(default=False)
```
Comment (kimberlythegeek) [inline treeherder/webapp/api/performance_serializers.py:463]: perhaps I'm missing something, but I don't see that `noise_metric` is being calculated anywhere. Did you intend to do this in a different patch?

Comment (beatrice-acasandrei) [reply treeherder/webapp/api/performance_serializers.py:463]: Yes, I'll add it in another PR, the logic is not straightforward for this and I needs to investigate the code some more, by default is false, and this is what I set here.

**Review by eng-esther — APPROVED** [review]

File: treeherder/webapp/api/performance_serializers.py (hunk 1 above)
Comment (kimberlythegeek) [reply treeherder/webapp/api/performance_serializers.py:463]: gotcha :+1:

**Review by kimberlythegeek — APPROVED** [review]
ran locally, and everything works as expected as far as I can tell. thanks!

Should we rename this PR and the ticket to remove 'is_noise_metric' since that will be covered in a later patch?
